src/thunderbots/software/ai/intent/intent_visitor.h

```diff
@@ -1,15 +1,15 @@
 #pragma once
 
-#include "ai/intent/catch_intent.h"
-#include "ai/intent/chip_intent.h"
-#include "ai/intent/direct_velocity_intent.h"
-#include "ai/intent/direct_wheels_intent.h"
-#include "ai/intent/dribble_intent.h"
-#include "ai/intent/kick_intent.h"
-#include "ai/intent/move_intent.h"
-#include "ai/intent/movespin_intent.h"
-#include "ai/intent/pivot_intent.h"
-#include "ai/intent/stop_intent.h"
+class CatchIntent;
```

Comment:
added:
```
// We forward-declare all the intents because if we include them we induce a 
// circular dependency between the Individual library for each intent and this
// visitor. Ex: `CatchIntent` includes `IntentVisitor`, but `IntentVisitor` 
// also includes `CatchIntent`

```